newIDE/app/src/InstancesEditor/CompactInstancePropertiesEditor/index.js

```diff
@@ -0,0 +1,217 @@
+          instance.getCustomProperties(project, layout),
+        (instance: gdInitialInstance, name, value) =>
+          instance.updateCustomProperty(name, value, project, layout)
+      );
+
+      const reorderedInstanceSchemaForCustomProperties = reorderInstanceSchemaForCustomProperties(
+        instanceSchemaForCustomProperties,
+        i18n
+      );
+
+      return {
+        object,
+        instanceSchema: is3DInstance
+          ? schemaFor3D.concat(reorderedInstanceSchemaForCustomProperties)
+          : schemaFor2D.concat(reorderedInstanceSchemaForCustomProperties),
+      };
+    },
+    [project, layout, instance, schemaFor2D, schemaFor3D, i18n]
+  );
+
+  if (!object || !instance || !instanceSchema) return null;
+
+  return (
+    <ErrorBoundary
+      componentTitle={<Trans>Instance properties</Trans>}
+      scope="scene-editor-instance-properties"
+    >
+      <ScrollView
+        autoHideScrollbar
+        key={instances
+          .map((instance: gdInitialInstance) => '' + instance.ptr)
+          .join(';')}
+      >
+        <Column expand noMargin id="instance-properties-editor">
+          <Column>
+            <CompactPropertiesEditor
+              unsavedChanges={unsavedChanges}
+              schema={instanceSchema}
+              instances={instances}
+              onInstancesModified={onInstancesModified}
+            />
+            <Separator />
+            <Line alignItems="center" justifyContent="space-between">
+              <Text size="sub-title" noMargin>
+                <Trans>Instance Variables</Trans>
+              </Text>
+              <IconButton
+                size="small"
+                onClick={() => {
+                  editInstanceVariables(instance);
+                }}
+              >
+                <ShareExternal fontSize="small" />
+              </IconButton>
+            </Line>
+          </Column>
+          {object ? (
+            <VariablesList
+              directlyStoreValueChangesWhileEditing
+              inheritedVariablesContainer={object.getVariables()}
+              variablesContainer={instance.getVariables()}
+              size="small"
+              onComputeAllVariableNames={() =>
+                object
+                  ? EventsRootVariablesFinder.findAllObjectVariables(
+                      project.getCurrentPlatform(),
+                      project,
+                      layout,
+                      object
+                    )
+                  : []
+              }
+              historyHandler={historyHandler}
+            />
+          ) : null}
+        </Column>
+      </ScrollView>
+    </ErrorBoundary>
+  );
+};
+
+const CompactInstancePropertiesEditorContainer = React.forwardRef<
+  Props,
+  CompactInstancePropertiesEditorInterface
+>((props, ref) => {
+  const forceUpdate = useForceUpdate();
+  React.useImperativeHandle(ref, () => ({
+    forceUpdate,
+  }));
+
+  return (
+    <Background minWidth={0}>
```

Comment:
I don't remember using a Background for a while. Is this a paper?

Reply:
I removed the background then!